esphome/components/pmsa003i/pmsa003i.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "esphome/core/component.h"
+#include "esphome/components/sensor/sensor.h"
+#include "esphome/components/i2c/i2c.h"
+
+namespace esphome {
+namespace pmsa003i {
+
+/**! Structure holding Plantower's standard packet **/
+// From https://github.com/adafruit/Adafruit_PM25AQI
+using PM25AQIData = struct PMSAQIdata {
```

Comment:
```suggestion
struct PM25AQIData {
```

(typedef/using struct not needed in C++)